examples/vault-iam-auth/auth-signature-scripts/sign-request.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
```

Comment:
Could you add a comment to this file explaining what it does and why it's in Python (as opposed to bash).

Reply:
Sure